mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -357,7 +357,11 @@ public MailboxSession createSystemSession(String userName) {
      */
 
     protected MailboxSession createSession(String userName, String password, SessionType type) {
-        return new SimpleMailboxSession(randomId(), userName, password, new ArrayList<>(), getDelimiter(), type);
+        return new SimpleMailboxSession(randomSessionId(), userName, password, new ArrayList<>(), getDelimiter(), type);
+    }
+
+    private MailboxSession.SessionId randomSessionId() {
```

Comment:
we don't care that it's random, the only thing we need to know is that it's a `new` Id

Reply:
renamed to `newSessionId()`